docs/Modpacks/Materials-and-Elements/The-ChemicalHelper-class.md

```diff
@@ -1,42 +1,76 @@
 ---
-title: The ChemicalHelper class
+title: ChemicalHelper
 ---
 
-It may behoove a packmaker working with GTCEu Modern to learn about the ChemicalHelper class. This class, available for use in server scripts, 
-contains a number of useful methods that can ease working with GTCEu Materials in contexts where it might not be possible, or it might be unsafe, to work with item or block tags.
+
+# The `ChemicalHelper` Class
+
+It may behoove a packmaker working with GTCEu Modern to learn about the ChemicalHelper class.
```

Comment:
> behoove

Listen, I love my lesser known words, but even I had to google what behoove meant :trolllaugh:
Probably just say its useful instead lol